docs/api/app.md

```diff
@@ -887,9 +887,9 @@ app.setJumpList([
 Returns `Boolean`
 
 This method makes your application a Single Instance Application - instead of
-allowing multiple instances of your app to run, this will ensure that only a
-single instance of your app is running, and other instances signal this
-instance and exit.
+allowing multiple instances of your app to run, this method obtains a lock which you
+ can then use to make your application single instance allowing for other 
```

Comment:
This isn't quite how I'd phrase it as the instant question will be "how can I use it".

To be honest I'd probably just delete this entire paragraph instead of trying to clarify it.  The next paragraph perfectly explains what the method does and how to use it.

Reply:
Sounds good 👍